src/client/auth/auth.proto

```diff
@@ -63,7 +63,21 @@ message ACL {
     map<string, Scope> entries = 1;
 }
 
+message User {
+    string username = 1;
+    bool admin = 2;
```

Comment:
I think it's cool that we include this in the User response, to save an etcd read. One risk though (maybe we just need to tell users, so they expect it?) is that if you revoke someone's admin access, they won't actually lose it until their token expires, which may be 24 hours later.